### DISCUSSION TITLE
[With Backend/ideas] 기본 repo의 README에 옮기진 repo 공지

### DISCUSSION THREAD

**Discussion by locodingve** [slamapp/server#20] [Ideas]
이번에 레포를 옮겼는데요. 
README에 레포가 옮겨 졌다는 내용과 위치를 공지해야할 것 같아서 discussion에 올려 봅니다.

**Comment by KwonYeKyeong** [comment]
아하!! 그러게요
이 작업은 이번주 토요일 스크럼때 같이 할까요??

**Reply by locodingve** [reply]
ㅋㅋㅋㅋㅋ 이번주 스크럼에 해요~~
